M13 release - documentation News (#174)
exoplatform/exo-documentation: docs/GettingStarted.rst

-  *Getting Started*: See a list of suggested actions you can start.
   See :ref:`Getting Started application <Getting Started>` for more details.

> -  *Calendar*: Get an overview of calendar events. See :ref:`Calendar application <CalendarApp>

Reviewer: seems to be unrelated to the news feature

Author: I didn't update this part.

Author: Fixed

|image198|

> When you click on the "See all" button,  you are redirected automatically to news application filtered by pinned articles

Reviewer 2: The changes on this sentence should be reverted IMO.

Author: fixed.

|image187|

> The page displays by default the list of the first 10 articles of the view "All News". You can load more articles using the "Load More" button.

Reviewer: ```suggestion
By default, the page displays the 10 latest articles of the  "All News" list. You can load more articles using the "Load More" button.
```

> The page displays by default the list of the first 10 articles of the view "All News". You have the possibility to load more articles using the appropriate button.

Reviewer: ```suggestion
The page displays by default the list of the first 10 articles of the view "All News". You can load more articles using the "Load More" button.
```

|image217|

> For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the information about the source space as well as the destination space(s) if the article has been shares.

Reviewer: ```suggestion
For each article you can see the title, the illustration, the first words of the summary or the article body, the author, the publication date, the number of views and the original space where the article was posted as well as the spaces where it was shared.
```

Different filters are available from News application so you can easily find articles posted in specific spaces, pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.

**Filter articles per spaces : ** You can search articles by spaces using the filter button from the top bar.

|image206|

> In the displayed drawer, you can choose from or search in the list of the spaces in which you are member the space(s) you want to filter by articles in the application then apply the using the button to be redirected to the filtered view. 

Reviewer 2: I think you revert the changes we discussed in the previous PR. Is it intended ?

Author: I didn't change this part. I created a new branch from master after doing a git fetch git pull. Will check

Author: Fixed

|image207|

|image208|

**Filter by pinned articles : ** To display the pinned news list, select this option from the dropdown menu of the application topbar.

|image197|

After applying this filter, the list will only contain the articles that have been pinned on the home page. You can reset the filters using the appropriate button inside the drawer.

**Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application topbar.

|image209|

> After applying this filter, the list will only display the articles posted by you. You have the possibility to archive, edit or share them directly from the application.

Reviewer: ```suggestion
After applying this filter, the list will only display the articles posted by you. You can either archive, edit or share them directly from the news list.
```

|image210|

**Filter by archived articles : ** To display your archived articles, select the option "Archived News" from the dropdown menu of the application topbar.

> After applying this filter, the list will only display your archived articles or all archived articles ifyou have the publisher role

Reviewer 2: ```suggestion
After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role
```

> After applying this filter, the list will only display your archived articles or all archived articles ifyou have the publisher role

Reviewer: ```suggestion
After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role
```

|image218|

Archiving News
---------------

As an article's author or as a platform-wide publisher (publisher:/platform/web-contributors role required), you can archive articles from News application interface.
An "archive/unarchive" icon is available in the right side of the article's preview next to share, edit and pin icons.

|image211|

When you click on the archive icon, a confirmation message appears

|image212|

> Once you confirm the action, the article is automatically moved into the view "Archived News" of the application and will be no longer available from the application to other users. Only the users having the role publisher (in addition to the author) will be able to see it in their archived view.

Reviewer: ```suggestion
Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the filter. Only the authors and publishers can see articles in the archive.
```

> The preview of the archived article activity from the feed will be changed :

Reviewer: ```suggestion
Archived articles have a different display in the activity stream  :
```

|image213|

> Other users will not be able to click on this preview as long as the article is archived. Even if they already have the article's URL, the details are no longer visible. The following message appears :

Reviewer: ```suggestion
Other users won't able to click on this preview as long as the article remains archived. Also, for them, accessing an archived article by URL won't let them read it. In such case, the following message appears :
```

|image214|

> On the other hand, you still have access to the article's details and you are able to unarchive it from this view.

Reviewer: ```suggestion
However, you, as the author (or publisher), keep have access to the article's details and you can unarchive it from there.
```

|image215|

> It is also possible to unarchive articles using the same icon from news application.

Reviewer: ```suggestion
The same icon allows to unarchive articles from the News application.
```

|image216|

> Once you confirm the unarchiving action. the article becomes available again to the other users.

Reviewer: ```suggestion
When you confirm the unarchiving of an item, it becomes accessible again to other users.
```

> .. note:: When you archive a pinned article it will be automatically unpinned from home page. 

Reviewer: ```suggestion
.. note:: When you archive a pinned article it gets automatically unpinned. However, if you unarchive the same article afterwards, it won't be re-pinned automatically. 
```

|image208|

**Filter by pinned articles : ** To display the pinned news list, select this option from the dropdown menu of the application topbar.

|image197|

After applying this filter, the list will only contain the articles that have been pinned on the home page. You can reset the filters using the appropriate button inside the drawer.

> **Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application topbar.

Reviewer: ```suggestion
**Filter by your posted articles : ** To display the articles written by you, select the option "My posted News" from the dropdown menu of the application's top bar.
```

|image209|

After applying this filter, the list will only display the articles posted by you. You can either archive, edit or share them directly from the news application.

|image210|

**Filter by archived articles : ** To display your archived articles, select the option "Archived News" from the dropdown menu of the application topbar.
After applying this filter, the list will only display your archived articles or all archived articles if you have the publisher role

|image218|

Archiving News
---------------

As an article's author or as a platform-wide publisher (publisher:/platform/web-contributors role required), you can archive articles from News application interface.

> An "archive/unarchive" icon is available in the right side of the article's preview next to share, edit and pin icons.

Reviewer: ```suggestion
An "archive/unarchive" icon is available on the right side of the article's preview next to "share", "edit" and "pin" icons.
```

|image211|

> When you click on the archive icon, a confirmation message appears

Reviewer: ```suggestion
When you click on the archive icon, a confirmation message appears : 
```

|image212|

> Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the filter. Only the authors and publishers can see articles in the archive.

Reviewer: ```suggestion
Once you confirm the action, the article is automatically moved into the archive and is no longer available to other users. The archive can be accessed from the "Archived News" entry in the dropdown. Only the authors and publishers can see articles in the archive.
```

Archived articles have a different display in the activity stream  :

|image213|

Other users won't able to click on this preview as long as the article remains archived. Also, for them, accessing an archived article by URL won't let them read it. In such case, the following message appears :

|image214|

> However, you, as the author (or publisher), keep have access to the article's details and you can unarchive it from there.

Reviewer: ```suggestion
However, you, as the author (or publisher), maintain access to the article's details and you can unarchive it from there.
```
